CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 # OverReact Changelog
+
+## 5.2.0
+- [#920] Support prop conversion in OverReact to simplify RMUI converted props
```

Comment:
#nit I think this changelog could be a little more specific and reference the added APIs
```suggestion
- [#920] Support conversion of prop values via annotations: `@ConvertProp`, `@convertJsMapProp`, `@convertJsRefProp`
```